Source/ACE/Entity/Landblock.cs

```diff
@@ -186,6 +186,23 @@ public void RemoveWorldObject(ObjectGuid objectId, bool adjacencyMove)
             }
         }
 
+        /// <summary>
+        /// get the WorldObject specified by objectID
+        /// </summary>
+        public WorldObject GetWorldObjectByGuid(ObjectGuid objectId)

```

Comment:
Remove this method.  This might be needed eventually, but for now it's a red flag for "you're doing it wrong."